var/spack/repos/builtin/packages/openblas/package.py

```diff
@@ -50,6 +50,17 @@ class Openblas(Package):
 
     patch('make.patch')
 
+    @property
+    def blas_libs(self):
+        shared = True if '+shared' in self.spec else False
+        return find_libraries(
+            ['libopenblas'], root=self.prefix, shared=shared, recurse=True
```

Comment:
In the old code both `lib` and `lib64` were searched for the library. I don't know much the internals of `openblas`, so I opted for this approach.